### PR TITLE
TRRA-219: Add number of days out and unit head to Actual Expense Report

### PR DESCRIPTION
**Pull request by aprigge**
Acceptance Criteria:

- [x] add number of days out and unit head to Actual Expense Report

### CODE REVIEW

File: terra/views.py [hunk 1]
```diff
@@ -678,12 +680,14 @@ def render_to_response(self, context, **response_kwargs):
                                 [
                                     v["subunit"].manager,
                                     actualexpense.treq.traveler.unit,
-                                    actualexpense.treq.traveler.uid,
+                                    actualexpense.treq.traveler.unit.manager,
+                                    str(actualexpense.treq.traveler.uid),
```
Comment (akohler) [inline terra/views.py:684]: @aprigge Why did this `traveler.uid` in line 684 get newly wrapped in `str()`, while the value in line 704 did not?  Should they be consistent?  Is the `str()` necessary?

**Review by akohler — COMMENTED** [review]
@aprigge Just the one minor question, otherwise no controversy :)

File: terra/views.py (hunk 1 above)
Comment (aprigge) [reply terra/views.py]: Thanks, updated now

**Comment by akohler** [comment]
Merged... now let's see if Travis actually builds the new docker file for deployment.

**Pull request merged**